openmicroscopy/openmicroscopy
# Remove commons-lang dependency

Remove usage of commons-lang: insight only
This was generating an error when using the omero-ij plugin in Fiji
see https://trac.openmicroscopy.org.uk/ome/ticket/12713

To test this PR:
 * Make sure the server starts

Fiji Test
 * Check that there is a jar under the jars folder commons-lang3-3.x
 * Add the omero-ij plugin to Fiji plugins folder. 
 * Check that you can connect to OMERO using the plugin

This PR replaces https://github.com/openmicroscopy/openmicroscopy/pull/3474
--no-rebase
